### DISCUSSION TITLE
Quick question re: after-open file handler

### DISCUSSION THREAD

**Discussion by Lord-Kamina** [gabime/spdlog#3207] [Q&A] [answered]
Say I want to use the the after-open  file handler callback for a sink with `std::fputs`, as per the example in README.

Should I take care to error-check the call to `std::fputs`, or the stream after its done, or will spdlog do that later on its own?

**Comment by tt4g** [comment, marked as answer]
spdlog does not check for errors after the callback call: https://github.com/gabime/spdlog/blob/e593f6695c6065e6b345fe2862f04a519ed484e0/include/spdlog/details/file_helper-inl.h#L52-L57
Handling of errors that occur in the callback should be handled in the callback.
